tests/e2e/test_code_duplicate_percentage.py

```diff
@@ -1,11 +1,11 @@
 from src.__main__ import generate_debt_report
+from os import path
 
 
 def test_code_duplication_percentage():
+    current_file_path = path.abspath(path.dirname(__file__))
     assert (
-        generate_debt_report(
-            "/home/dev-06/Desktop/tinkin/python/you-owe-me/tests/repository-fixture/"
-        )
+        generate_debt_report(path.join(current_file_path, "../demo-repository"))
```

Comment:
Con estas lineas estas generando el siguiente path: `/home/sathors/work/tinkin/you-owe-me/tests/e2e/../demo-repository`, así que realmente no estás usando un path relativo, pero un absoluto.
Si no te quieres complicar la vida, puedes usar un path relativo a la raíz de nuestro proyecto mismo (desde donde corre el pytest).

```python
def test_code_duplication_percentage():
    assert (
        generate_debt_report('tests/demo-repository')
        == """
    Report Type      | Result
    -----------------|-----------
    Code Duplication | 40%
    """
    )
```

No te parece más sencillo?

Reply:
Si, esta mucho mas fácil y entendible, Gracias !!!!